Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY types types
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
 RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o pie ./cmd
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
Should the Ubuntu version of each yaml file's `runs-on:` in .github/workflows/ also be updated to 22.04?

Reply:
Thanks! I fixed it.